src/DynamoCoreWpf/Commands/SearchCommands.cs

```diff
@@ -48,5 +48,28 @@ public DelegateCommand HideSearchCommand
             }
         }
 
+        private DelegateCommand importLibrary;
+        public DelegateCommand ImportLibraryCommand
+        {
+            get
+            {
+                if (importLibrary == null)
+                    importLibrary = new DelegateCommand(this.dynamoViewModel.ImportLibrary,
+                        this.dynamoViewModel.CanImportLibrary);
+                return importLibrary;
+            }
+        }
+
+        private DelegateCommand showPackageManagerSearch;
+        public DelegateCommand ShowPackageManagerSearchCommand
+        {
+            get
+            {
+                if (showPackageManagerSearch == null)
+                    showPackageManagerSearch = new DelegateCommand(this.dynamoViewModel.ShowPackageManagerSearch,
+                        this.dynamoViewModel.CanShowPackageManagerSearch);
+                return showPackageManagerSearch;
+            }
+        }
```

Comment:
All is done (such commands have been there before) except one thing. I didn't change modifiers for `ImportLibrary`, `CanImportLibrary` as they were not touched in this PR.